<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICManualOperationPageFrame</class>
 <widget class="QFrame" name="ICManualOperationPageFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="zAxisToolButton">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="xAxisToolButton">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="yAxisToolButton">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="cAxisToolButton">
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="aAxisToolButton">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="bAxisToolButton">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="x2AxisToolButton">
       <property name="text">
        <string>X2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="y2AxisToolButton">
       <property name="text">
        <string>Y2</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="centralFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>350</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>403</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="functionToolButton">
       <property name="text">
        <string>Function</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="alternateIOToolButton">
       <property name="text">
        <string>Alternate IO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="fixtureToolButton">
       <property name="text">
        <string>Fixture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="suckerToolButton">
       <property name="text">
        <string>Sucker</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
